run.py

```diff
@@ -94,6 +105,49 @@ def config_validator():
         sys.exit(TOKEN_HELP)
 
 
+def initialize_status(statuses):
+    """Map the status name with the milestone id on GitHub.
+
+    The project needs the mapping in between milestones and their id
+    to be able to query status related things. It uses a backup version
+    when the request to GitHub fails.
+    """
+    print('Statuses Initialization…')
+    REPO_ROOT = app.config['ISSUES_REPO_URI'].rpartition('/issues')[0]
```

Comment:
Reminder: We should probably create a config or global variable for this, in all the various places we do these things. For future cleanup.